auther.go

```diff
@@ -62,17 +62,11 @@ func newAuther(config *Config) *auther {
 // request (temporary credential) according to RFC 5849 2.1.
 func (a *auther) setRequestTokenAuthHeader(req *http.Request) error {
 	oauthParams := a.commonOAuthParams()
-
```

Comment:
These changes fix the tests, but should have no effect on LTI since LTI doesn't use Access Tokens or Request Tokens.